### PR TITLE
PolarChart: fix series drawn incorrectly after force re-render (T1174645)

### PR DESCRIPTION
**Pull request by AlexanderMoiseev**

### CODE REVIEW

File: packages/devextreme/testing/tests/DevExpress.viz.charts/polarChart.tests.js [hunk 1]
```diff
@@ -739,3 +739,31 @@ QUnit.test('Change of useSpiderWeb', function(assert) {
     assert.strictEqual(stubAxes[2], chart.getArgumentAxis());
     assert.strictEqual(stubAxes[3], chart.getValueAxis());
 });
+
+QUnit.module('series', () => {
+    QUnit.test('point angle should not change after force render (T1174645)', function(assert) {
+        let pointAngle;
+
+        const chart = $($('#chartContainer')).dxPolarChart({
```
Comment (Krijovnick) [inline packages/devextreme/testing/tests/DevExpress.viz.charts/polarChart.tests.js]: `$('#chartContainer').dxPolarChart({...`

File: packages/devextreme/testing/tests/DevExpress.viz.charts/polarChart.tests.js [hunk 2]
```diff
@@ -739,3 +739,31 @@ QUnit.test('Change of useSpiderWeb', function(assert) {
...
+                { id: 1, arg: 10, val: 100 },
+                { id: 2, arg: 10, val: 100 },
+                { id: 3, arg: 20, val: 200 },
+            ],
+            series: {
+                type: 'line'
+            }
+        }).dxPolarChart('instance');
+
+        assert.strictEqual(pointAngle, 90, 'angle is correct after first render');
```
Comment (Krijovnick) [inline packages/devextreme/testing/tests/DevExpress.viz.charts/polarChart.tests.js]: You can move assertions into the onDrawn method.
And I think we should write the test for 'area' series.

File: packages/devextreme/testing/tests/DevExpress.viz.charts/polarChart.tests.js [hunk 3]
```diff
@@ -739,3 +739,31 @@ QUnit.test('Change of useSpiderWeb', function(assert) {
     assert.strictEqual(stubAxes[2], chart.getArgumentAxis());
     assert.strictEqual(stubAxes[3], chart.getValueAxis());
 });
+
+QUnit.module('series', () => {
```
Comment (alexanderPolosatov) [inline packages/devextreme/testing/tests/DevExpress.viz.charts/polarChart.tests.js]: let's make name of module more concrete

File: packages/devextreme/testing/tests/DevExpress.viz.charts/polarChart.tests.js [hunk 4]
```diff
@@ -739,3 +739,31 @@ QUnit.test('Change of useSpiderWeb', function(assert) {
...
+                { id: 3, arg: 20, val: 200 },
+            ],
+            series: {
+                type: 'line'
+            }
+        }).dxPolarChart('instance');
+
+        assert.strictEqual(pointAngle, 90, 'angle is correct after first render');
+
+        chart.render({ force: true });
```
Comment (alexanderPolosatov) [inline packages/devextreme/testing/tests/DevExpress.viz.charts/polarChart.tests.js]: let's separate this test to 2 (do not add new act section after arrange)

**Review by alexanderPolosatov — CHANGES_REQUESTED** [review]

**Review by Krijovnick — APPROVED** [review]

**Review by alexanderPolosatov — APPROVED** [review]

**Pull request merged**